src/cript/api/__init__.py

```diff
@@ -3,3 +3,6 @@
 from cript.api.api import API
 from cript.api.valid_search_modes import SearchModes
 from cript.api.vocabulary_categories import VocabCategories
+
+# Default maximum seconds for all requests to the backend
+_API_TIMEOUT = 25
```

Comment:
how come we are adding this?

Reply:
The new linter tool `bandit` is complaining if you don't have a time out.
And I think they have a good point, so adding the new version of trunk adds this.

Reply:
I think the API and HTTP times out at some point anyways and I think some of the search like bigsmiles search takes a while for the response to come back which might create issues if our timeout kicks in sooner than it has time to respond.

Did we have bandit before?

Reply:
No Bandit is new.
We don't have a timeout set before. And that means, that connections might hang forever.

It is good practice to set one:
https://requests.readthedocs.io/en/latest/user/advanced/#timeouts